

## Conclusion

Secondary metabolites biosynthesis impulsed under the action of phytohormone in general and jasmonates in particular is crucial for the plant growth, development and defense in a stressed environment. So far, tremendous efforts drive to the elucidation of some key genes involved into the physiological mechanisms deployed by the plant to grow and survive. Owing the growing interest for the natural produced secondary metabolites in human health [ref1] and neutraceutical enriched foods [ref 2], a better comprehension of the accumulation of the plant metabolites is gaining more attention.  The jasmonate-elicited plant secondary metabolism machinery involved several transcription factors including WRKY TFs.

Huge progress have been done by using some popular plant system including oryza sativa [ref1], arabidopsis thaliana [ref2], nicotiana tabbacum [ref3] etc.However, the orphan crops also represent a gold mine that deserve to be investigated. Among those orphan crop, the transcriptional comprehension of secondary metabolites synthesis have been initiated [ref1, ref2, ref3], paving the way for not only effective health oriented metabolic enginering, but also engineered crop protection against adverse biotic and abiotic stresses. Altogether, under the ongoing environmental change status, JAs' metabolites elicitation machinery needs further investigation.
















## New Ref for transcriptomics


[CRISPR cas 9](https://www.mdpi.com/2223-7747/10/10/2055)



-01 2020 | Imbibition of Seed Germination  | [RNA-Seq Study Reveals AP2-Domain-Containing Signalling Regulators Involved in Initial Imbibition of Seed Germination in Rice](https://www.sciencedirect.com/science/article/pii/S1672630820300408)

-02 2017 | Coleoptile growth | [RNA-Seq Analysis of Diverse Rice Genotypes to Identify the Genes Controlling Coleoptile Growth during Submerged Germination](https://www.frontiersin.org/articles/10.3389/fpls.2017.00762/full)

-03 2016 | Seed germination | [Transcriptome Analysis of Oryza sativa (Rice) Seed Germination at High Temperature Shows Dynamics of Genome Expression Associated with Hormones Signalling and Abiotic Stress Pathways](https://link.springer.com/article/10.1007/s12042-016-9170-7)

-04 2010 | Seed dormancy | [Transcriptomics Analysis Identified Candidate Genes Colocalized with Seed Dormancy QTLs in Rice ( Oryza sativa L.)](https://link.springer.com/content/pdf/10.1007/s12374-010-9120-0.pdf)

-05 2021 | [Integrating GWAS and Transcriptomics to Identify Genes Involved in Seed Dormancy in Rice](https://assets.researchsquare.com/files/rs-414570/v1/90cf3036-41a4-44d0-bdb4-a06bb621104a.pdf?c=1631881054)

-06 2020| PHS | [Analysis of Varietal Differences in Pre-harvest Sprouting of Rice using RNA-Sequencing](https://www.koreascience.or.kr/article/JAKO202034965719330.page)


## New ref for GWAS

-01 [Genome-wide association mapping revealed a diverse genetic basis of seed dormancy across subpopulations in rice (Oryza sativa L.)](https://bmcgenomdata.biomedcentral.com/articles/10.1186/s12863-016-0340-2)

-02 2021 | [Novel Sources of Pre-Harvest Sprouting Resistance for Japonica Rice Improvement](https://www.mdpi.com/2223-7747/10/8/1709/htm)


Excellent summary

> In the case of rice, the map-based cloning of qSD7-1/qPC1 identified the Rc gene, encoding a basic helix-loop-helix family transcription factor, which upregulates genes involved in the biosynthesis of both flavonoid and ABA [12]. In the T-DNA/Tos17 insertional mutant populations, genes involved in the biosynthesis of carotenoids (i.e., OsPDS, OsZDS, OsCRTISO, and β-OsLCY), the important precursors of ABA, were identified [13]. A similar study using rice mutants with the PHS phenotype identified causal mutations in OsCNX1 and OsCNX6, the genes encoding molybdenum cofactors required for ABA biosynthesis [14]. Also, characterization of phs9-D revealed that this PHS mutant in rice carries a dominant mutation in PHS9, a gene encoding a CC-type glutaredoxin, which mediates ABA signaling through the interaction with ABA receptors [15]. In spite of these findings underlying clear biochemical mechanisms on PHS resistance, breeding efforts enhancing PHS resistance in rice varieties have been limited due to lack of genetic resources. Sdr4 and qSD7-1/qPC1, the major loci associated with PHS resistance were from a single genetic resource: Aus landrace Kasalath having red pericarp color and strong dormancy [12,16].



## New for genome wide identification


01 2021 | [Genome-Wide Identification and Expression Analysis of OsbZIP09 Target Genes in Rice Reveal Its Mechanism of Controlling Seed Germination](https://www.mdpi.com/2223-7747/10/8/1709/htm)




## New functional validation 

-01 [Control of rice pre-harvest sprouting by glutaredoxin-mediated abscisic acid signaling](https://onlinelibrary.wiley.com/doi/full/10.1111/tpj.14501)












# Review
   

Genomics assisted breeding for PHS has retained the attention of the scientific community the last two decades. Tremendous advances were carried out with the identification of useful PHS resistance genes. A glance of the diverse strategies employed for PHS resistance detection is summarized in the [*figure 1*](https://github.com/Yedomon/auxin_repressed_gene/blob/main/Figurev2.png). The first and foremost stage in this process relied on the characterization of germplasm for the identification of promising genetics resources. 


![img](https://github.com/Yedomon/auxin_repressed_gene/blob/main/Figurev2.png)
**Figure 1: Diagrammatic of the genomics-assisted breeding for PHS tolrance in rice.**


**QTL**

Meanwhile several parental lines were released including IR24, Asominori, Jinsang, and Gopum. Besides the detection of contrasting materials regarding PHS resistance, the resistance gene pool from wild relatives including O. rufipogon [(Phan et al., 2021)](https://link.springer.com/article/10.1007/s42976-021-00162-0#ref-CR20) and O. nivara [(Li et al., 2006)]( https://nph.onlinelibrary.wiley.com/doi/full/10.1111/j.1469-8137.2005.01647.x) and weedy rice [Gu et al 2004](https://academic.oup.com/genetics/article/166/3/1503/6050471?login=true#236962316) was investigated. Several mapping populations were generated following RILs, BILs, CSSLs, Three-way cross hybrid, and double haploidy.  The earlier detection of QTLs relied mostly on RFLP and SSRs markers. However, whole genome sequencing approach with a high resolution mapping at SNP level has been initiated. A total of xxxx QTLs have been detected within all chromosomes of the rice genome *(Table 1)*. 

**Comparative genomics**

At early stage of PHS gene detection, comparative genomics has been tested. Orthologous genes identification by comparing with well characterized PHS genes in barley and wheat exhibited the presence of the hormonal GA20-oxidase encoding gene [Li et al. 2004]( https://link.springer.com/article/10.1007%2Fs10142-004-0104-3). 


**Transcriptomics**


In order to find out genes involved in PHS and subsequent related biological processes including seed dormancy and maturation, transcriptome studies ([Huh et al., 2013]( https://www.sciencedirect.com/science/article/abs/pii/S0176161713001053),[Xie et al., 2019](https://bp.ueb.cas.cz/pdfs/bpl/2019/01/35.pdf), [Park et al., 2021](https://www.frontiersin.org/articles/10.3389/fpls.2021.727302/full)) using contrasting rice materials has been performed. Interestingly, candidate genes regulating hormones including ABA, GA and IAA have been highlighted including transcription factors such as DREB (dehydration-responsive element-binding protein), Basic Helix-Loop-Helix Transcription Factor (bHLH), late embryogenesis abundant protein (LEA), NAC transcription factor, and CCAAT-HAP3 transcription factor. This emphasizes the contributions of TFs as one of the key players mediating of the hormonal expression. 

**RNA splicing**

Meanwhile microRNAs (miRNAs) are well known to be dependent of hormonal regulation in planta [(Reyes et al., 2007)](https://onlinelibrary.wiley.com/doi/10.1111/j.1365-313X.2006.02980.x)). Recently, [Park et al. (2021)]( https://www.frontiersin.org/articles/10.3389/fpls.2021.727302/full#h4) identified two candidate miRNAs (osa-miR5827 and osa-miR1862h) associated with two PHS-related genes  OsFbox594 and OsbHLH084 respectively. 


**GWAS and Genome-wide identification**

High quality genome resource enabled the detection of PHS related-genes via genome wide association analysis and genome-wide identification approaches. [Zhu et al. (2021)](https://www.mdpi.com/1422-0067/22/4/1661) pinpointed the role of a bZIP transcription factor OsbZIP09 which expression is induced by ABA. The mutation of this gene inhibited PHS in rice. By mining GWAS and transcriptome data, [Shi et al. (2021)](https://link.springer.com/article/10.1007/s00122-021-03911-1) found a significant effect of the variation of SNPs in the promoter region of the Os9BGlu33  gene regarding germination index. 


**Functional validation**

From those huge genetic and genomic resources, few genes have been functionally validated. [(Sugimoto et al., 2010)](https://www.pnas.org/content/107/13/5792.short) identified the Sdr4 gene as responsible of seed dormancy control. Interestingly, OsVP1 exhibited a regulatory effect on Sdr4 gene via ABA signaling pathway. [(Chen et al., 2021)](https://www.sciencedirect.com/science/article/pii/S2214514120301045). Transcription factor also play a crucial role for regulating the hormonal expression in rice. [Hobo et al. (1999)](https://www.pnas.org/content/96/26/15348.short), [Wu et al. (2021)](https://onlinelibrary.wiley.com/doi/epdf/10.1111/pbr.12848) and [Wang et al. 2020](https://pubs.acs.org/doi/abs/10.1021/acs.jafc.0c04748) demonstrated the interaction between VP1 and TRAP1 (bZIP transcription factor) and Rc (basic helix–loop–helix (bHLH) transcription factor) genes for ABA regulation. 

**Perspectives**

Considering the importance of environmental effect on seed dormancy and germination, we suggest an investigation of the epigenome on rice. In fact, an increasing evidence has been in favor of DNA and histone methylation in regards with PHS resistance genetic mechanism [(Singh et al. 2013)](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0077009), [Lujan-Soto et al 2021](https://www.mdpi.com/2223-7747/10/2/236). The role of the ARGONAUTE4_9, a DNA methylation RNA-dependent gene has been proved in wheat PHS. However, the epigenetic framework of PHS in rice is still elusive. Therefore, deciphering the epigenetic factors contributing to the PHS resistance regulation in rice will lay a foundation for a deep understanding of the full machinery in real climate impacted conditions.  Moreover, an intensive validation of the existing candidate genes should be processed via Crispr-Cas9, RNAi, Agrobacterium tumefaciens mediated transformation for example. This will surely boost the design of PHS resistant rice.
Altogether, post-transcriptional regulation encompassing, splicing RNA and epigenetics offer a new avenue for unravelling of the mechanism of resistance of PHS in rice. Ultimately, a deeper comprehension of the whole machinery will provide a gain for designing better rice with added value.













# auxin_repressed_gene





[tr1](https://bp.ueb.cas.cz/pdfs/bpl/2019/01/35.pdf)

[tr2](https://www.frontiersin.org/articles/10.3389/fpls.2021.727302/full)

[tr3](https://www.sciencedirect.com/science/article/abs/pii/S0176161713001053)

[tr4](https://www.frontiersin.org/articles/10.3389/fpls.2017.02213/full)

[tr5]()



[Genome-Wide Association Study for Pre-harvest Sprouting Resistance in a Large Germplasm Collection of Chinese Wheat Landraces](https://www.frontiersin.org/articles/10.3389/fpls.2017.00401/full)


[Genome-Wide Identification and Comparative Analysis of ARF Family Genes in Three Apiaceae Species](https://www.frontiersin.org/articles/10.3389/fgene.2020.590535/full)



# transcriptome pre sprouting rice


[Genetic map locations for orthologous Vp1 genes in wheat and rice](https://link.springer.com/content/pdf/10.1007/s001220051069.pdf)



## Generalities 

- [Cereal comparative genetics and preharvest sprouting](https://link.springer.com/article/10.1023/A:1019675723265)

- 2005 [Dormancy release, ABA and pre-harvest sprouting](https://www.sciencedirect.com/science/article/pii/S1369526605000154)

- 2021 very nice review 2021 [Pre-harvest sprouting in cereals: genetic and biochemical mechanisms](https://academic.oup.com/jxb/article-abstract/72/8/2857/6104898?redirectedFrom=fulltext)

[Abscisic acid and the pre-harvest sprouting in cereals](https://www.tandfonline.com/doi/full/10.4161/psb.3.12.6606)

## Identification of valuable resource for pre-sprouting resistence


[Variation in pre-harvest sprouting resistance, seed germination and changes in abscisic acid levels during grain development in diverse rice genetic resources](https://www.cambridge.org/core/journals/plant-genetic-resources/article/variation-in-preharvest-sprouting-resistance-seed-germination-and-changes-in-abscisic-acid-levels-during-grain-development-in-diverse-rice-genetic-resources/5255A3A1176672E1914C5ADB0CF3A1E2)

[Novel Sources of Pre-Harvest Sprouting Resistance for Japonica Rice Improvement](https://www.mdpi.com/2223-7747/10/8/1709)


## Genome wide identification area


[Genome-Wide Identification and Expression Analysis of OsbZIP09 Target Genes in Rice Reveal Its Mechanism of Controlling Seed Germination](https://www.mdpi.com/1422-0067/22/4/1661)





## Oriza sativa

### Transcriptomics

[Comparative transcriptome profiling of developing caryopses from two rice cultivars with differential dormancy](https://www.sciencedirect.com/science/article/pii/S0176161713001053)




[Identification of Genes and MicroRNAs Affecting Pre-harvest Sprouting in Rice (Oryza sativa L.) by Transcriptome and Small RNAome Analyses](https://www.frontiersin.org/articles/10.3389/fpls.2021.727302/full)

[Analysis of Varietal Differences in Pre-harvest Sprouting of Rice using RNA-Sequencing](https://www.cropbio.or.kr/articles/pdf/DG4Q/kjcs-2020-065-04-0.pdf)

[The RNA-seq transcriptome analysis identified genes related to rice seed dormancy](https://bp.ueb.cas.cz/pdfs/bpl/2019/01/35.pdf)

### Agronomy area

- [Pyramiding seed dormancy genes to improve resistance of semi-dwarf varieties to pre-harvest sprouting in rice](https://link.springer.com/article/10.1007/s11032-020-01172-2)


### GWAS


[Integrating GWAS and Transcriptomics to Identify Genes Involved in Seed Dormancy in Rice](https://assets.researchsquare.com/files/rs-414570/v1/12bf1792-e835-43c5-b506-902da87afc9d.pdf?c=1618612249)


### QTL detection




- 2003 [Identification of quantitative trait loci associated with pre-harvest sprouting resistance in rice (Oryza sativa L.)](https://www.sciencedirect.com/science/article/pii/S0378429002002174)

[Detection of Whole-Genome Resequencing-Based QTLs Associated with Pre-Harvest Sprouting in Rice (Oryza sativa L.)](https://www.plantbreedbio.org/journal/view.html?uid=656&&vmd=Full)

[Genetic dissection of pre-harvest sprouting resistance in an upland rice cultivar](https://www.jstage.jst.go.jp/article/jsbbs/68/2/68_17062/_html/-char/en)


## Oriza japonica


[QTL mapping for pre-harvest sprouting resistance in japonica rice varieties utilizing genome re-sequencing](https://link.springer.com/article/10.1007/s00438-020-01688-4)


[Detection of quantitative trait loci controlling pre-harvest sprouting resistance by using backcrossed populations of japonica rice cultivars](https://link.springer.com/article/10.1007/s00122-010-1275-z)

[QTL mapping for pre-harvest sprouting resistance in japonica rice varieties utilizing genome re-sequencing](https://link.springer.com/article/10.1007/s00438-020-01688-4)


## Functionnal validation Oryza sativa

- 2008 [Mutations of genes in synthesis of the carotenoid precursors of ABA lead to pre-harvest sprouting and photo-oxidation in rice](https://onlinelibrary.wiley.com/doi/full/10.1111/j.1365-313X.2008.03411.x)

[OsVP1 activates Sdr4 expression to control rice seed dormancy via the ABA signaling pathway](https://www.sciencedirect.com/science/article/pii/S2214514120301045)
[Control of rice pre-harvest sprouting by glutaredoxin-mediated abscisic acid signaling](https://onlinelibrary.wiley.com/doi/10.1111/tpj.14501)

[Molecular cloning of Sdr4, a regulator involved in seed dormancy and domestication of rice](https://www.pnas.org/content/107/13/5792.short)

[A bZIP factor, TRAB1, interacts with VP1 and mediates abscisic acid-induced transcription](https://www.pnas.org/content/96/26/15348.short)

[Transcription Factors Rc and OsVP1 Coordinately Regulate Preharvest Sprouting Tolerance in Red Pericarp Rice](https://pubs.acs.org/doi/abs/10.1021/acs.jafc.0c04748)



### Contribution of wild relative



[Improving the pre-harvest sprouting resistance of rice cultivar IR36 using wild rice (Oryza rufipogon) W630](https://link.springer.com/article/10.1007/s42976-021-00162-0)


### ABA regulation

> Recently, key regulators of ABA biosynthesis or signaling pathway have been characterized: LEC2, ABI3, and DOG1 in Arabidopsis [[11], [12], [13], [14]]; TaPHS1 and TaMFT in wheat [2,15]; VP1 in maize [16]; and OsFbx352, OsNCED3, OsPDS, OsCRTISO, β-OsLCY, OsZDS, OsVP1, Sdr4, OsDSG1, OsABI3, OsABI5, PHS8, PHS9, and OsMFT2 in rice [1,7,[17], [18], [19], [20], [21], [22], [23], [24], [25], [26], [27], [28], [29], [30]]. All of these genes control seed dormancy, but their regulatory mechanisms remain elusive.


## Wheat case

[Identification of QTLs and a Candidate Gene for Reducing Pre-Harvest Sprouting in Aegilops tauschii–Triticum aestivum Chromosome Segment Substitution Line](Identification of QTLs and a Candidate Gene for Reducing Pre-Harvest Sprouting in Aegilops tauschii–Triticum aestivum Chromosome Segment Substitution Lines)


## Comaparative genomics for identification of candidate genes


[Genes controlling seed dormancy and pre-harvest sprouting in a rice-wheat-barley comparison](https://link.springer.com/article/10.1007%2Fs10142-004-0104-3)

## Simple research on germination

[Dynamic transcriptome and metabolome analyses of two types of rice during the seed germination and young seedling growth stages](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-07024-9)










